site/src/docs/getting-started/tutorials/migrate-to-4.0.0.mdx

```diff
@@ -195,75 +195,95 @@ There are updates to lineheights to improve readability especially on Headings.
+- **[Headings]** Line heights have been updated to improve readability. Please check that the layout does not break.
+    
+- **[Form components]** The asterisk indicating required fields in form components like TextInput has been renewed and is now its own property. You’ll need to manually change the required indicators in layouts.
+    
+- **[Tag]** Accessibility has been improved, and a new Link variant has been introduced. These changes do not break the old components, but it’s still better to check the updated components.
+    
 
 ### What should I do?
-1. Duplicate your current design file
-In order to avoid disruptions to your workflow, it is recommended that you duplicate your project files before accepting this update. This allows you to migrate to the latest HDS Design Kit while maintaining a reference to your original file.
 
-2. Review and accept updates
-In your duplicate file, review and accept the HDS Design Kit library updates. You will see updates to three libraries which include the 
+#### 1. Duplicate your current design file
+
+In order to avoid disruptions to your workflow, it is recommended that you [duplicate](https://help.figma.com/hc/en-us/articles/360038511533-Duplicate-or-copy-files) your project files before accepting this update. This allows you to migrate to the latest HDS Design Kit while maintaining a reference to your original file.
+
+#### 2. Review and accept updates
 
-3. Find deprecated dropdown components
-You can now easily highlight deprecated components. Read more in section Figma major version release process | 📣 Introduction of Figma Variables in HDS.
+In your duplicate file, review and accept the HDS Design Kit library updates. You will see updates to three libraries which include the
+
+#### 3. Find deprecated dropdown components
+
+- You can now easily highlight deprecated components. Read more in section [Introduction of Figma Variables in HDS](#introduction-of-figma-variables-in-hds).
+    
+#### 4. Update components & content
 
-5. Update components & content
 This update includes breaking changes to components and may require that you update content in components with breaking changes.
 
 ### How can I continue using HDS v3?
+
 The v3 Figma library has been copied as a new file and will no longer receive updates. However, you can continue to use it by swapping to the v3 library.
 
 Swapping libraries allows you to quickly update instances in a file to use components from another library. Follow the Figma guide to swap to the v3 library if you wish to use the previous HDS version.
 
-Swap style and component libraries – Figma Learn - Help Center
-
+- [Swap style and component libraries – Figma Learn - Help Center](https://help.figma.com/hc/en-us/articles/4404856784663-Swap-style-and-component-libraries#swap)
+    
 ### What about the Sketch library?
+
 The Sketch library will no longer be updated and will be removed after the 4.0.0 release.
 
+---
```

Comment:
I would not separate this section with a line, since separation is also used between Dev and design changes on this guide and the variables section still pertains to Figma. 

How about putting it before ###Getting support ?